src/components/IssueUI/IssueRequestStatusUI/ManualIssueExecutionUI/index.tsx

```diff
@@ -0,0 +1,136 @@
+import clsx from 'clsx';
+import { useTranslation } from 'react-i18next';
+import { useMutation, useQuery, useQueryClient } from 'react-query';
+import { toast } from 'react-toastify';
+
+import { displayMonetaryAmount } from '@/common/utils/utils';
+import InterlayDenimOrKintsugiMidnightOutlinedButton from '@/components/buttons/InterlayDenimOrKintsugiMidnightOutlinedButton';
+import ErrorModal from '@/components/ErrorModal';
+import { WRAPPED_TOKEN, WRAPPED_TOKEN_SYMBOL } from '@/config/relay-chains';
+import { ISSUES_FETCHER } from '@/services/fetchers/issues-fetcher';
+import { TABLE_PAGE_LIMIT } from '@/utils/constants/general';
+import { QUERY_PARAMETERS } from '@/utils/constants/links';
+import { getColorShade } from '@/utils/helpers/colors';
+import useQueryParams from '@/utils/hooks/use-query-params';
+
+interface Props {
+  request: {
+    id: string;
+    backingPayment: {
+      amount: number;
+      btcTxId: string;
+    };
+    vault: {
+      accountId: string;
+      collateralToken: {
+        token: CurrencyIdLiteral;
+      };
+      wrappedToken: {
+        token: CurrencyIdLiteral;
+      };
+    };
+  };
+}
+
+const ManualIssueExecutionUI = ({ request }: Props): JSX.Element => {
+  const { t } = useTranslation();
+
+  const queryParams = useQueryParams();
+  const selectedPage = Number(queryParams.get(QUERY_PARAMETERS.PAGE)) || 1;
+  const selectedPageIndex = selectedPage - 1;
+
+  const queryClient = useQueryClient();
+
+  // TODO: should type properly (`Relay`)
+  const executeMutation = useMutation<void, Error, any>(
+    (variables: any) => {
+      if (!variables.backingPayment.btcTxId) {
+        throw new Error('Bitcoin transaction ID not identified yet.');
+      }
+      return window.bridge.issue.execute(variables.id, variables.backingPayment.btcTxId);
+    },
+    {
+      onSuccess: (_, variables) => {
+        queryClient.invalidateQueries([ISSUES_FETCHER, selectedPageIndex * TABLE_PAGE_LIMIT, TABLE_PAGE_LIMIT]);
+        toast.success(t('issue_page.successfully_executed', { id: variables.id }));
+      }
+    }
+  );
+
+  const { data: vaultCapacity, error: vaultCapacityError } = useQuery({
+    queryKey: 'vault-capacity',
+    queryFn: async () => {
+      const vaultAccountId = newAccountId(window.bridge.api, request.vault.accountId);
+
+      const issue = await window.bridge.issue.getRequestById(request.id);
+
+      const collateralToken = await currencyIdToMonetaryCurrency(
+        window.bridge.assetRegistry,
+        issue.vaultId.currencies.collateral
+      );
+
+      return await window.bridge.vaults.getIssuableTokensFromVault(vaultAccountId, collateralToken);
+    }
+  });
+
+  // TODO: should type properly (`Relay`)
+  const handleExecute = (request: any) => () => {
+    executeMutation.mutate(request);
+  };
+
+  const backingPaymentAmount = newMonetaryAmount(request.backingPayment.amount, WRAPPED_TOKEN);
+
+  const executable = vaultCapacity?.gte(backingPaymentAmount);
```

Comment:
This is a bug. When the issue request is in `PendingWithEnoughConfirmations` then the request is always executable. Only if the issue request is `Cancelled`.